### PR TITLE
[1342] Tweaks

### PR DESCRIPTION
**Pull request by rfb**

### CODE REVIEW

**Comment by rfb** [comment]
[EPIC#AquiferData | Sorting Search Results |1| !!!](https://trello.com/c/wccxmu5A/1328-epicaquiferdata-sorting-search-results-1)

**Comment by rfb** [comment]
[EPIC#AquiferData | Sorting Search Results |1| !!!](https://trello.com/c/wccxmu5A/1328-epicaquiferdata-sorting-search-results-1)

**Comment by rfb** [comment]
[EPIC#AquiferData | Tweaks |2| !!!](https://trello.com/c/l97uqAtt/1342-epicaquiferdata-tweaks-2)

File: app/frontend/src/aquifers/components/Search.vue [hunk 1]
```diff
@@ -16,19 +16,21 @@
   <b-card no-body class="p-3 mb-4">
     <h5>Aquifer Search</h5>
```
Comment (Sybrand) [inline app/frontend/src/aquifers/components/Search.vue]: @stephenhillier we're going to have to assess these tags throughout the site at some point. (re. accessibility)

**Pull request merged**